homewok_20/Main.java

```diff
@@ -0,0 +1,21 @@
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+public class Main {
+    public static void main(String[] args) {
+        PetrolStation petrolStation = new PetrolStation(150);
+
+        FuelRequests fuelRequests1 = new FuelRequests(petrolStation, 20);
+        FuelRequests fuelRequests2 = new FuelRequests(petrolStation, 30);
+        FuelRequests fuelRequests3 = new FuelRequests(petrolStation, 50);
+
+        ExecutorService executorService = Executors.newFixedThreadPool(3);
```

Comment:
executor service should be shut down gracefully, you can use try-with-resources or `executorService.shutdown()`